classes/class-wc-connect-api-client.php

```diff
@@ -529,7 +529,7 @@ protected function authorization_header() {
 			}
 
 			list( $token_key, $token_secret ) = explode( '.', $token->secret );
-			$token_key                        = sprintf( '%s:%d:%d', $token_key, JETPACK__API_VERSION, $token->external_user_id );
+			$token_key                        = sprintf( '%s:%d:%d', $token_key, defined( 'JETPACK__API_VERSION' ) ? JETPACK__API_VERSION : 'embed', $token->external_user_id );
```

Comment:
The format is `%s:%d:%d` so the second and third placeholders should be numbers (i.e. can't be "embed"). Testing on my local, it looks like the code below returns `1` anyway so I suggest we hardcode it to that, and in the future use Jetpack's header building function instead of our own, if possible.

```php
Constants::get_constant( 'JETPACK__API_VERSION' )
```